README.md

```diff
@@ -1,20 +1,20 @@
-1. BUILD ParaView (known to work on Linux with ParaView 5.4.1)
-2. BUILD Fesapi (with static linking on HDF5 libraries) v0.13.0.0 : https://github.com/F2I-Consulting/fesapi/archive/v0.13.0.0.tar.gz
+1. BUILD Fesapi (with static linking on HDF5 libraries) v0.16.0.0 : https://github.com/F2I-Consulting/fesapi/archive/v0.16.0.0.tar.gz
```

Comment:
We could also use dynamic linking on HDF5 libraries. But in the latter case, ParaView must be built with same HDF5 libraries.